src/libs/dbHandler.go

```diff
@@ -1,11 +1,57 @@
 package libs
 
 import (
+	"database/sql"
 	"errors"
+	"time"
 
+	"github.com/DATA-DOG/go-sqlmock"
 	"github.com/accuknox/auto-policy-discovery/src/types"
 )
 
+// ======== //
+// == DB == //
+// ======== //
+
+// DBHandle : To store DB handle
+var DBHandle *sql.DB = nil
+var MockSql sqlmock.Sqlmock = nil
+var MockDB *sql.DB = nil
+
+// InitDB : Initilaize DB driver
+func InitDB(cfg types.ConfigDB) {
+	if cfg.DBDriver == "mysql" {
+		DBHandle = ConnectMySQL(cfg)
+	} else if cfg.DBDriver == "sqlite3" {
+		DBHandle = ConnectSQLite(cfg)
+	}
+}
+
+// WaitForDB
+func WaitForDB(db *sql.DB) {
+	for {
+		err := db.Ping()
+		if err != nil {
+			time.Sleep(time.Second * 1)
+			log.Error().Msgf("DB Ping() failed. Will retry. err=%s", err.Error())
+		} else {
+			break
+		}
+	}
+}
+
+func NewMockDB() (*sql.DB, sqlmock.Sqlmock) {
+	db, mock, err := sqlmock.New()
```

Comment:
Why to add this function is `dbHandler.go` when it is used only from `_test` file?

Reply:
@nyrahul This function existed in both DB files. Hence maintained in the same location.

Reply:
But according to this commit, this is a newly added function?